buildSrc/src/main/groovy/io.micronaut.build.internal.spring-module.gradle

```diff
@@ -2,6 +2,11 @@ plugins {
     id 'io.micronaut.build.internal.module'
 }
 
+repositories {
```

Comment:
Create a base module to reduce duplication. see https://github.com/micronaut-projects/micronaut-aws/pull/1445 and https://github.com/micronaut-projects/micronaut-project-template/tree/master/buildSrc/src/main/groovy 

We could apply `base` module for the `test-suite`